pkgs/build-support/appimage/default.nix

```diff
@@ -7,11 +7,17 @@
 , pv
 , squashfsTools
 , buildFHSUserEnv
-, pkgs
+, runCommand
+, substituteAll
 }:
 
 rec {
-  appimage-exec = pkgs.substituteAll {
+  # for compatibility, deprecated. TODO: find a clean way to add these to top-level/aliases.nix
+  extractType1 = extract;
+  extractType2 = extract;
+  wrapType1 = wrapType2;
```

Comment:
This is not possible with top-level/aliases.nix but we can use ``config.allowAliases``.

Reply:
Updated the comment. I'll use `allowAliases` in a future PR when I have time to switch over the existing appimage derivations.